ccmlib/cluster.py

```diff
@@ -444,19 +444,26 @@ def run_cli(self, cmds=None, show_output=False, cli_options=None):
             raise common.ArgumentError("No live node")
         livenodes[0].run_cli(cmds, show_output, cli_options)
 
-    def set_configuration_options(self, values=None, batch_commitlog=None):
+    def set_configuration_options(self, values=None):
```

Comment:
We should let this take `**kwargs`, no? So other callers don't need to change? Though we're already changing behavior out from under them, so maybe we should make the error loud